tests/cases/config.yaml

```diff
@@ -89,15 +89,17 @@ Environments:
       facility_devices_user2_pwd: add_your_info  
       facility_devices_user2_number: add_your_info
 #Employee users    
-      employee_user1: add_your_info
-      employee_user1_pwd: add_your_info
-      employee_user1_name: add_your_info
-      employee_user1_lastname: add_your_info
-      employee_user1_dob: add_your_info
-      employee_user1_card_member_id: add_your_info
+      employee_user1: dguzman+employee1@neural.com.do
```

Comment:
can we change the values to "add_your_info" and make a comment next to the phone number and dob placeholders with the format of the value?